autoload/fzf/vim.vim

```diff
@@ -712,7 +712,7 @@ endfunction
 
 function! s:get_git_root(dir)
   let dir = len(a:dir) ? a:dir : substitute(split(expand('%:p:h'), '[/\\]\.git\([/\\]\|$\)')[0], '^fugitive://', '', '')
-  let root = systemlist('git -C ' . fzf#shellescape(dir) . ' rev-parse --show-toplevel')[0]
+  let root = fzf#with_sh({ -> systemlist('git -C ' . fzf#shellescape(dir) . ' rev-parse --show-toplevel')[0] })
```

Comment:
Would it help to just not use `fzf#shellescape` here?

```suggestion
  let root = systemlist('git -C ' . shellescape(dir) . ' rev-parse --show-toplevel')[0]
```

Reply:
You are right. It fixes the issue. Do you want me to push this change and close the other PR?

Reply:
Yes, this seems to be the simplest and more to the point.

I can understand the confusion around `fzf#shellescape`. It was inteded to be used when building a command that is going to be run by fzf.

Reply:
OK. Thanks for the clarification. I indeed got confused about this.